src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -14,6 +14,9 @@ import { Dropdown } from "components/atoms/Dropdown";
 
 import "./SpacesDropdown.scss";
 
+const noneOptionName = "None";
+const spaceNoneOption = { none: { name: "", template: undefined } };
```

Comment:
This is a mutable object passed to a component which after few renders might or might not pass it some place else or modify it on its own.

Yes, with a close inspection one might determine that is not the case, but there is a quick and easy way of marking it as actual constant: `Object.freeze()` i.e. something like:

```suggestion
const spaceNoneOption = Object.freeze({ none: Object.freeze({ name: "", template: undefined }) } );
```
or like:
```suggestion
const spaceNoneOption = { none: { name: "", template: undefined } };
Object.freeze(spaceNoneOption);
Object.freeze(spaceNoneOption.none);
```

Reply:
The above is just a note, I don't think it merits holding back the PR, just for future consideration

Reply:
done ef728b1